exercises/armstrong-numbers/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Get the exercise name from the current directory
```

Comment:
I made a change to the default CMakeLists.txt file after you made this. See #276. Can you update this file?